package.json

```diff
@@ -44,6 +44,11 @@
       "build": {
         "dependsOn": [
           "^build"
+        ],
+        "outputs": [
+          "dist/**",
+          ".next/**",
+          "!.next/cache/**/*"
```

Comment:
this was tricky to figure out, but if packages/apps have special output folders they need to be listed